.travis.yml

```diff
@@ -2,12 +2,16 @@ language: android
 android:
   components:
     - build-tools-22.0.1
-    - android-19
-    - android-16
+    # old build tools due to core lib
+    - build-tools-20.0.0
+    - android-22
+    # Additional components
+    - extra-google-m2repository
+    - extra-android-m2repository
 before_install:
   - rm pom.xml
 script:
-  - ./setup_env.sh ant
```

Comment:
The problem is the (old) exploded aar appcompat in /libs (version 19 instead auf 22.*) which would now be needed. I couldn't find any documentation on how to create the exploded aar folder sturcture. In case this is resolved the ant build should work again.